.github/workflows/action.yml

```diff
@@ -95,6 +95,8 @@ jobs:
         config:
           - version: 7.x
             flags: --incompatible_enable_proto_toolchain_resolution
+          - version: 8.x
+            flags: --incompatible_enable_proto_toolchain_resolution
 
     runs-on: ubuntu-latest
     steps:
```

Comment:
With bazel 6 gone from the bzlmod test matrix I think it's better to put this flag into e2e/bazel-bzlmod-toolchain-from-source/.bazelrc, but we can merge without this change, it's more around dropping some tech debt

Reply:
Done